### ISSUE TITLE
redux intro only?

### ISSUE THREAD

**Issue by austinsamsel** [kucharskimaciej/devmeetings-react-redux-workshop#2]
I noticed the Redux portion was trimmed and that the tasks are no longer included.

https://github.com/kucharskimaciej/devmeetings-react-redux-workshop/commit/d7ccd59b25b04dd4fc3c4afa181d691364bb7127

Are we not presenting Redux tasks to students anymore?

**Comment by sscaff1** [comment]
I'm presenting in LA on July 15th. I can walk though a simple implementation of redux with the class. I don't think it has be added to the slides. What do you guys think?

**Comment by kucharskimaciej** [comment]
Sorry guys, didn't mean to go behind your back with that.
We tested the workshop twice and decided to replace the Redux portion with the intro to ES2015, as this was a problem for participants. There was usually very little time left for Redux anyways, so we want to do another workshop on Redux alone.

The content is there as a teaser; I did like a 5-10 minutes intro on it during the last module.
